utils/graphql/resolvers.ts

```diff
@@ -0,0 +1,90 @@
+import * as loader from "./loaders";
```

Comment:
this file feels unnecessary... those are one line functions so it's easier to just see them directly on the schema.ts

Reply:
for now it's true.
But some of them could be used to rename keys - the keys that the organic Hashavshevet endpoints return varies from this apps keys. For example Hash's "batchNo" is this app's "batch_id". Another example is changing data order logic (keys that return different types of data, or data for different entities).
This is where that magic would happen once I'll get to it